File: Service_eHospital_DongNai_A/Implement/XacNhanChiPhi_DELService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class XacNhanChiPhi_DELService : BaseService<XacNhanChiPhi_DEL, IXacNhanChiPhi_DELRepository>
    , IXacNhanChiPhi_DELService
    {
    private readonly IXacNhanChiPhi_DELRepository _XacNhanChiPhi_DELRepository;
    public XacNhanChiPhi_DELService(IXacNhanChiPhi_DELRepository XacNhanChiPhi_DELRepository) : base(XacNhanChiPhi_DELRepository)
    {
    _XacNhanChiPhi_DELRepository = XacNhanChiPhi_DELRepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A/XacNhanChiPhi_DEL.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { XacNhanChiPhi_DEL } from './XacNhanChiPhi_DEL.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class XacNhanChiPhi_DELService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'ID', 'Ma_File', 'Ma_Lk', 'XacNhanChiPhi_Id', 'TiepNhan_Id', 'BenhAn_Id', 'BenhNhan_Id', 'SoBHYT', 'Loai', 'NgayVao', 'NgayRa', 'NgayDel', 'isExport'];

    List: XacNhanChiPhi_DEL[] | undefined;
    ListFilter: XacNhanChiPhi_DEL[] | undefined;
    FormData!: XacNhanChiPhi_DEL;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "XacNhanChiPhi_DEL";
    }
}

